Course/Course/Presentation/Handouts/HandoutsView.swift

```diff
@@ -107,6 +107,22 @@ struct HandoutsView: View {
                         }
                     }
                 )
+                
+                // MARK: - Error Alert
+                if viewModel.showError {
+                    VStack {
+                        Spacer()
+                        SnackBarView(message: viewModel.errorMessage)
+                    }
+                    .padding(.bottom, viewModel.connectivity.isInternetAvaliable
+                             ? 0 : OfflineSnackBarView.height)
+                    .transition(.move(edge: .bottom))
+                    .onAppear {
+                        doAfter(Theme.Timeout.snackbarMessageLongTimeout) {
+                            viewModel.errorMessage = nil
+                        }
+                    }
+                }
```

Comment:
I guess this isn't required and you can remove `showError` from `HandoutsViewModel`.